src/main/java/jalau/at18/azul/Tiles.java

```diff
@@ -0,0 +1,21 @@
+package jalau.at18.azul;
+
+public enum Tiles {
```

Comment:
For class and enum names, it would make sense to use names in singular name instead of plural, this because when we create an instance, this represents a single object and not multiple. Normally, plural names make more sense for variable names that are data structure types
```
enum Tile{} // represents a tile
Tile redTile = Tile.RED // create a tile
List<Tile> tiles = new List<Tile> // a list of tiles
```